doc/api/module.md

```diff
@@ -317,6 +317,37 @@ isBuiltin('fs'); // true
 isBuiltin('wss'); // false
 ```
 
+### `module.mapCallSite(callSite)`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+> Stability: 1.1 - Active development
+
+* `callSite` {Object | Array} A [CallSite][] object or an array of CallSite objects.
+* Returns: {Object | Array} The original source code location(s) for the given CallSite object(s).
+
+Reconstructs the original source code location from a [CallSite][] object through the source map.
+
+```mjs
+import { mapCallSite } from 'node:module';
+import { getCallSite } from 'node:util';
```

Comment:
Since `getCallSite` is in `node:util`, doesn't it make sense to move it to `node:util` as well? It looks out of place.

Reply:
All the sourcemap related APIs are in `node:module` so it makes sense to be there